typings/index.d.ts

```diff
@@ -1165,6 +1165,7 @@ declare module 'discord.js' {
     public readonly member: GuildMember | null;
     public status: PresenceStatus;
     public readonly user: User | null;
+    public userID: string | number;
```

Comment:
```suggestion
    public userID: Snowflake;
```

This should be typed as being a snowflake as with other id typings, also, ids are always strings, never number